### PR TITLE
Sp02 ranking  page

### PR DESCRIPTION
**Pull request by Rayane0823**

### CODE REVIEW

File: frontend/src/components/Homepage/Homepage.css [hunk 1]
```diff
@@ -81,26 +60,23 @@ p {
...
 
-img {
+.logo-home {
   width: auto;
   height: 15rem;
 }
 
-@media screen and (min-width: 480px) {
-  img {
+@media screen and (max-width: 480px) {
```
Comment (Yoann-Fortin) [inline frontend/src/components/Homepage/Homepage.css]: Mobile-first always

File: frontend/src/components/Homepage/Homepage.css [hunk 2]
```diff
@@ -120,19 +96,12 @@ img {
...
-    padding-top: 0;
-    margin: 0.2rem;
-  }
 }
 
 a {
   text-decoration: none;
 }
 
 .OpenedPopUp {
```
Comment (Yoann-Fortin) [inline frontend/src/components/Homepage/Homepage.css:105]: No PascalCase in CSS

File: frontend/src/components/RankingPage/RankingPage.css [hunk 3]
```diff
@@ -0,0 +1,286 @@
+.BackgroundRanking {
```
Comment (Yoann-Fortin) [inline frontend/src/components/RankingPage/RankingPage.css]: No PascalCase

File: frontend/src/components/RankingPage/RankingPage.css [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+
+.LastRank {
+  font-size: 2rem;
+}
+
+.NoDisplay {
+  display: none;
+}
+
+@media screen and (max-width: 720px) {
```
Comment (Yoann-Fortin) [inline frontend/src/components/RankingPage/RankingPage.css]: Mobile first always

File: frontend/src/components/RankingPage/RankingPage.css [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+    background: silver;
+    grid-area: 2 / 1 / 7 / 8;
+  }
+  .RankingLineFirst:nth-child(3) {
+    background: #cd7f32;
+    grid-area: 3 / 15 / 8 / 22;
+  }
+}
+
+@media screen and (max-width: 530px) {
```
Comment (Yoann-Fortin) [inline frontend/src/components/RankingPage/RankingPage.css]: Mobile first always

File: frontend/src/components/RankingPage/RankingPage.jsx [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+        </table>
+        <section className="RankingContainer">
+          <section className="ThreeFirstsBloc ThreeFirstsBlocResponsive">
+            {sortedRanking.map((ranking, index) => (
+              <section
+                className={index < 3 ? "ThreeFirstsCards" : "NoDisplay"}
+                key={ranking.rank}
+              >
+                <h3 className="ThreeFirstsName">{ranking.rank}</h3>
+                <h1 className="ThreeFirstsName">{ranking.name}</h1>
```
Comment (Yoann-Fortin) [inline frontend/src/components/RankingPage/RankingPage.jsx]: Only h1 by page

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/RankingPage/RankingPage.css [hunk 7]
```diff
@@ -33,6 +32,17 @@
   align-items: center;
 }
 
+.ThreeFirstsBloc {
```
Comment (Yoann-Fortin) [inline frontend/src/components/RankingPage/RankingPage.css]: No PascalCase in CSS

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/RankingPage/RankingPage.css [hunk 8]
```diff
@@ -13,6 +13,7 @@
 .RankingTitle {
```
Comment (Yoann-Fortin) [inline frontend/src/components/RankingPage/RankingPage.css]: No PascalCase

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by SebSamPro — APPROVED** [review]

**Review by JulienOza — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**